app/models/tenant.rb

```diff
@@ -1,7 +1,3 @@
 class Tenant < ApplicationRecord
   validates :external_tenant, :uniqueness => true, :presence => true
-
-  def self.tenancy_enabled?
-    ENV["BYPASS_TENANCY"].blank?
```

Comment:
Same comment as above. Is this something we're wanting to get rid of?

Reply:
@lindgrenj6 Yes, technically there is no reason we would ever run without tenancy enabled.  This was partially done so we could work on transitioning pre-existing test over to a tenanted world.